innovation_sweet_spots/pipeline/pilot/investment_predictions/create_dataset/create_dataset.py

```diff
@@ -128,14 +125,13 @@ def create_dataset(
     cb_wrangler = CrunchbaseWrangler()
 
     # Load datasets
+    nrows = 20_000 if test else None
```

Comment:
It's really good to have a test mode, thanks! Very minor point, but perhaps you could add `test` to the final filename in those cases, so that one doesn't overwrite the full datasets when experimenting?